apps/editor.planx.uk/src/pages/FlowEditor/components/forms/FormModal.tsx

```diff
@@ -119,7 +159,7 @@ const FormModal: React.FC<{
   extraProps?: any;
 }> = ({ type, handleDelete, Component, id, before, parent, extraProps }) => {
   const { navigate } = useNavigation();
-  const formikRef = useRef<FormikProps<any> | null>(null);
+  const formikRef = useRef<FormikProps<BaseNodeData & unknown> | null>(null);
```

Comment:
I missed that this was an `any` when reviewing https://github.com/theopensystemslab/planx-new/pull/5871